Dispatcher/Controllers/Homepage/HomeViewController.swift

```diff
@@ -94,6 +127,13 @@ extension HomeViewController: UITableViewDataSource {
         return 40
     }
     
+    private func formatDate(_ date: String) -> String {
```

Comment:
It's better practice to create this function inside a util file because we might need this function elsewhere and we don't want to write the same code more than once